dbadmin/migrations/006_alter_cluster_table_for_archive_db_writer.up.sql

```diff
@@ -0,0 +1,7 @@
+GRANT SELECT, INSERT, UPDATE, DELETE ON cluster TO archive_db_writer;
+ALTER TABLE cluster ALTER COLUMN status DROP NOT NULL;
+ALTER TABLE cluster ALTER COLUMN version DROP NOT NULL;
+ALTER TABLE cluster ALTER COLUMN cve_cache_critical DROP NOT NULL;
```

Comment:
not necessary to drop `NOT NULL` for these `cve_cache_*`, they have `DEFAULT 0`, so they will always have value

Reply:
true